### DISCUSSION TITLE
Is there anyway to track successful downloads?

### DISCUSSION THREAD

**Discussion by kjgarza** [plausible/analytics#1394] [Anything Else]
currently, I have been looking at how to track event clicks and I do that to track file downloads (with custom events). However, that doesn't mean that the downloads are successful (the wrong link, something stopped the download, etc). Any idea of how can I make a setup to track successful downloads? 

Best

**Comment by metmarkosaric** [comment]
that's not possible to do through web analytics. our script needs to be inserted into the header of any page you'd like to track but downloads such as files / pdfs etc don't allow you to insert any javascript. what you're doing with custom events is the best way to get this type of data using web analytics. something like server logs might give you more insights into actual file downloads rather than just clicks on the file downloads

**Comment by metmarkosaric** [comment]
You can now track clicks on files automatically with Plausible! All the most common file extensions are supported but you can also specify custom file types to track. Here's how to enable file tracking https://plausible.io/docs/file-downloads-tracking
